cyclops/models/catalog.py

```diff
@@ -61,19 +61,19 @@ def decorator(model_obj: type) -> type:
         _model_catalog[name] = model_obj
 
         if model_type == "static":
-            _static_model_keys.add(model_obj.__name__)
+            _static_model_keys.add(name)
```

Comment:
Oh, I see. I changed it because the names we used for registering the models matched the names we used for the model configs. This came up when I tried to use the model registry name to get the model's config file.
Creating a mapping would work. The alternative is to rename the config files to match the model class name.

Reply:
Let's go with creating a mapping.

Reply:
I will add that when I update the tasks.